colossalai/fx/graph_module.py

```diff
@@ -0,0 +1,140 @@
+import os
+import warnings
+import torch
+import torch.nn as nn
+from torch.nn.modules.module import _addindent
+from torch.fx.graph_module import GraphModule, _EvalCacheLoader, _WrappedCall, _exec_with_source, _forward_from_src
```

Comment:
Change this to the following code to avoid import error with different PyTorch versions.
```python
try:
     from torch.fx.graph_module import xxx
except:
    pass
```

Reply:
Okay, and to avoid mistakes in torch11, I currently skip the test in our CI, and make our `ColoGraphModule` the same as `GraphModule` if we use torch11.

```python
try:
    from torch.fx.graph_module import GraphModule, _EvalCacheLoader, _WrappedCall, _exec_with_source, _forward_from_src
    from torch.fx.graph import Graph, _PyTreeCodeGen, _is_from_torch, _custom_builtins, PythonCode
    COLOGM=True
except:
    from torch.fx.graph_module import GraphModule
    from torch.fx.graph import Graph
    pass
    COLOGM=False
```